diracx-routers/pyproject.toml

```
@@ -56,6 +56,8 @@ auth = "diracx.routers.auth:router"
 WMSAccessPolicy = "diracx.routers.job_manager.access_policies:WMSAccessPolicy"
 SandboxAccessPolicy = "diracx.routers.job_manager.access_policies:SandboxAccessPolicy"
 
+[project.entry-points."diracx.min_client_version"]
+diracx = "diracx.routers:DIRACX_MIN_CLIENT_VERSION"
```

Any reason not to write directly the version in `pyproject.toml`:

```suggestion
diracx = "0.1.0"
```

It sounds like it would be easier to bump the version from a CI job (e.g. based on the project version) for instance if it is written in `pyproject.toml` rather than in a variable in the code.

---

Yes, the reason is that you cannot write string directly as value of an "entry-points". 
From what I understand, if wa want a custom metadata we can use "[tool.diracx.min_client_version]" but in that case we can't retreive the value using "importlib.metadata".